### PR TITLE
Update to winit 0.18.0 + release glutin 0.19.0

### PR DESCRIPTION
**Pull request by francesca64**
This also updates any dependencies that were updated in winit.

### CODE REVIEW

**Comment by Eijebong** [comment]
Anything blocking on that ??

File: Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "glutin"
 version = "0.18.0"
```
Comment (Eijebong) [inline Cargo.toml]: It's missing the version bump btw :p

Comment (francesca64) [reply Cargo.toml]: Whoops, thanks! Good catch.

**Comment by francesca64** [comment]
Ah, sorry! I forgot to merge this.

**Pull request merged**